src/sagemaker/local/image.py

```diff
@@ -106,6 +106,8 @@ def train(self, input_data_config, output_data_config, hyperparameters, job_name
         data_dir = self._create_tmp_folder()
         volumes = self._prepare_training_volumes(data_dir, input_data_config, output_data_config,
                                                  hyperparameters)
+        # If local, source directory needs to be updated to mounted /opt/ml/code path
+        hyperparameters = self._update_local_src_path(hyperparameters, key=sagemaker.estimator.DIR_PARAM_NAME)
```

Comment:
an integ test is failing because it expects this hyperparameter to be JSON (at least for the SageMaker deep learning framework images)

Reply:
Ah good catch! Updated hyperparameters with json conversion

Reply:
Unable to run integration tests on my machine, so please let me know if there are any more errors on those tests